### ISSUE TITLE
[Bug]: Update all not checking the checkbox for mods with missing files

### ISSUE THREAD

**Issue by miklkit** [KSP-CKAN/CKAN#4207]
### Is there an existing issue for this?

- [X] I have checked the existing issues for an existing issue. I even actually looked at the ones that are pinned to the top!

### Operating System

Windows 11

### CKAN Version

CKAN v1.35.1.24271

### Games

KSP 1

### Game Version

KSP 1.12.5.3190

### Did you make any manual changes to your game folder (i.e., not via CKAN)?

Yes I have added updates to OPT Spaceplanes via Github.

### Describe the bug

CKAN wanted to update, so I let it.  Then it wanted to update something but when I clicked on the update button it flashed but nothing changed.

### Steps to reproduce

It does this every time after restarting CKAN now.  The hilited mod is Kerbal Inventory Systems

**Comment by HebaruSan** [comment]
Hi @miklkit, thanks for the report.
What happens if you check the Update checkbox yourself?

**Comment by miklkit** [comment]
Hi!

  Now that was interesting.  I just clicked the box to update KIS, hit the "apply" button, and it updated.   Then the top line was replaced by MechJeb, so I clicked on that and then "apply" and it updated.  Then it all cleared up and is normal now.  I just clicked on the "refresh" button and now the "update" column is gone too.  Just learned a little trick if it happens again.

  miklkit
-----Original Message-----
